### PR TITLE
Add suffix_bits for TsoResponse

### PR DESCRIPTION
**Pull request by JmPotato**
Signed-off-by: JmPotato <ghzpotato@gmail.com>

### CODE REVIEW

File: proto/pdpb.proto [hunk 1]
```diff
@@ -121,6 +121,7 @@ message TsoRequest {
 message Timestamp {
     int64 physical = 1;
     int64 logical = 2;
+    uint32 suffix_bits = 3;
```
Comment (AndreMouche) [inline proto/pdpb.proto:124]: Could you please add some annotation for this param?

Comment (JmPotato) [reply proto/pdpb.proto:124]: Nice idea.

**Review by Yisaer — APPROVED** [review]
LGTM

**Review by nolouch — APPROVED** [review]

**Pull request merged**